calico.go

```diff
@@ -316,9 +316,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 			fmt.Fprintf(os.Stderr, "Calico CNI creating profile: %s\n", conf.Name)
 			var inboundRules []api.Rule
 			if wepIDs.Orchestrator == "k8s" {
```

Comment:
nit (and out of scope):  would be nice to update this to use apis.v3.OrchestratorKubernetes

but wouldn't let that stop us merging.